lib/internal/errors.js

```diff
@@ -1207,7 +1207,9 @@ E('ERR_UNHANDLED_ERROR',
 E('ERR_UNKNOWN_BUILTIN_MODULE', 'No such built-in module: %s', Error);
 E('ERR_UNKNOWN_CREDENTIAL', '%s identifier does not exist: %s', Error);
 E('ERR_UNKNOWN_ENCODING', 'Unknown encoding: %s', TypeError);
-E('ERR_UNKNOWN_FILE_EXTENSION', 'Unknown file extension: %s', TypeError);
+E('ERR_UNKNOWN_FILE_EXTENSION',
+  'Unknown file extension %s for %s imported from %s',
```

Comment:
We should quote the file extension here as well, in case there is no file extension it will display `''` then.

Reply:
Sounds good, added that